arch/arm64/core/Kconfig

```diff
@@ -28,6 +25,12 @@ config CPU_CORTEX_A72
 	help
 	  This option signifies the use of a Cortex-A72 CPU
 
+config HAS_ARM_SMCCC
+	bool
+	help
+	  Include support for the Secure Monitor Call (SMC) and Hypervisor
+	  Call (HVC) instructions on Armv7 and above architectures.
```

Comment:
armv7?

Reply:
Yes, this is valid from armv7 up.

Reply:
I thought arm64 started on armv8, but maybe my memory is wrong.

Reply:
What I mean is that SMCCC is a standard valid also for armv7, hence the description in the help. But in Zephyr ARMv8 is the only to actually use that.